### ISSUE TITLE
issue for training

### ISSUE THREAD

**Issue by ShutongLiang** [Marshall-mk/MedMamba#1]
Have you encountered any difficulties during the training process? For example, issues with high computational resource usage or slow training? The Mamba architecture is known for its efficiency, but training with MedMamba is extremely slow, and the final results are not satisfactory.

**Comment by Marshall-mk** [comment]
Yes, training MedMamba is extremely slow and requires very high compute. I
couldn't use more than 8 batch size on Google collab's gpu.
